converter_app/converters.py

```diff
@@ -180,7 +180,7 @@ def match_value(self, identifier, value):
                 return value if value else False
 
             else:  # match == 'exact'
-                result = (value == identifier.get('value'))
+                result = (value == str(identifier.get('value')))
```

Comment:
Maybe we should use `value_str` or something similar throughout the converter to indicate that a value is a string for sure.